tests/conftest.py

```diff
@@ -1,5 +1,4 @@
 import pytest
-
```

Comment:
Muss hier nicht die Leerzeile bleiben?

Reply:
Hm, ehrlich gesagt, weiß ich gar nicht genau, wie da die reine Lehre ausschaut. Habe es gerade noch einmal ausprobiert, `isort` beschwert sich bitterlich, wenn da eine Leerzeile ist. 
` Imports are incorrectly sorted and/or formatted.`

Reply:
:-D Dann lassen wir es einfach so...